### DISCUSSION TITLE
Different way to show errors

### DISCUSSION THREAD

**Discussion by Ssionn** [zed-industries/zed#13592] [Feature Requests]
Is there a different way of showing the errors instead of this list? it's making me pretty confused
![image](https://github.com/zed-industries/zed/assets/90753599/a346f7aa-1239-493d-aff5-6fbe3dca4517)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
